### ISSUE TITLE
Wrong attribute values in request after last update

### ISSUE THREAD

**Issue by badgersow** [a85/POSTMan-Chrome-Extension#2218]
1. Postman Version: 4.5.0
2. App (Chrome app or Mac app): Mac app
3. OS details: Darwin 95.108.173.176-red.dhcp.yndx.net 15.6.0 Darwin Kernel Version 15.6.0: Thu Jun 23 18:25:34 PDT 2016; root:xnu-3248.60.10~1/RELEASE_X86_64 x86_64
4. Is the Interceptor on and enabled in the app: No
5. Did you encounter this recently, or has this bug always been there: After today update
6. Expected behaviour: Param will be passed in query exactly as it was set
7. Console logs (http://blog.getpostman.com/2014/01/27/enabling-chrome-developer-tools-inside-postman/ for the Chrome App, View->Toggle Dev Tools for the Mac app):
8. Screenshots (if applicable)

![2016-08-02_13-25-58](https://cloud.githubusercontent.com/assets/13203977/17325587/7559dde6-58bd-11e6-9cb5-0900b05bf6f7.png)

Type param value as in screenshot above and send request.
Here is what my machine gets:

```
graev:~$ nc -l 1234
GET /?param=(key HTTP/1.1
cache-control: no-cache
Postman-Token: e95afefd-e992-4dfb-95aa-b89cfc972e67
User-Agent: PostmanRuntime/2.2.5
host: localhost:1234
accept-encoding: gzip, deflate
Connection: keep-alive
```

Here is expected behavior (as it was in versions before):

```
graev:~$ nc -l 1234
GET /?param=(key==value) HTTP/1.1
cache-control: no-cache
Postman-Token: e95afefd-e992-4dfb-95aa-b89cfc972e67
User-Agent: PostmanRuntime/2.2.5
host: localhost:1234
accept-encoding: gzip, deflate
Connection: keep-alive
```

**Comment by abhijitkane** [comment]
@czardoz 
`(new sdk.Url('http://httpbin.org?a=(c==d)')).getRaw()` returns `"http://httpbin.org?a=(c"`

**Comment by czardoz** [comment]
@badgersow Thanks for reporting this, the bug has been fixed, and should be out in the next release :)

**Comment by badgersow** [comment]
Many thanks, @czardoz.

When this release should be available?
Is there any workaround right now?

**Comment by badgersow** [comment]
@abhijitkane @czardoz 

Guys, could you please answer to my comment above?

**Comment by shamasis** [comment]
It's out in v4.5.1.

**Comment by shamasis** [comment]
Additional fixes for this is going out in v4.6 tentatively scheduled for Mac app today. Will close the issue once it is released. Confirmation of the fix when it is received would be helpful. 😊

**Comment by badgersow** [comment]
After `4.5.1` everything was ok for me.
Thanks!
